tests/test_scene_optimizer.py

```diff
@@ -42,7 +41,6 @@ def test_create_computation_graph(self):
             # create dask client
             cluster = LocalCluster(n_workers=1, threads_per_worker=4)
 
-            matching_regime = ImageMatchingRegime.EXHAUSTIVE
```

Comment:
nit: if we need to change this, can we add an inline comment here that we are using sequential matching by default? otherwise it's a little opaque imho

Reply:
Nevermind, seeing now that we are specifying this by the retriever only.